src/data/faq.json

```diff
@@ -118,7 +118,6 @@
             },
             {
                 "title": "Vaccination Cert.",
-                "id": "vac_cert",

```

Comment:
Wrong line?

Reply:
That's embarassing. Fixed in https://github.com/corona-warn-app/cwa-website/pull/1342/commits/dbbe111e244081953e120be824fa4d0f764428de